README.org

```diff
@@ -997,7 +1006,7 @@ into their own module in ~treemacs-icons.el~).
 If you want to delve into the treemacs' code base, check out [[https://github.com/Alexander-Miller/treemacs/wiki][the wiki]] for some general pointers.
 
 * Dependencies
- - emacs >= 26.1
+ - emacs >= 26.1 (>= 27.1 for tab-bar)
```

Comment:
New module will need a new tag line for commit messages. Let's go with `[Tab Bar]` on this one.